src/util/createResponsiveStoreEnhancer.js

```diff
@@ -1,17 +1,30 @@
+// third party imports
+import assign from 'lodash/assign'
 // local imports
 import addEventHandlers from './addEventHandlers'
 
 /**
  * Creates a store enhancer based off an (optional) throttle time.
- * @arg {number} [throttleTime=100] - Throttle time (in miliseconds) for the
- * window resize event handler.
+ * @arg {object} [options={throttleTime,calculateStateInitially}] - Options object.
+ * @arg {number} [options.throttleTime=100] - Throttle time (in miliseconds) for
+ * the window resize event handler.
+ * @arg {boolean} [options.calculateStateInitially=true] - True if the responsive
+ * state must be calculated initially, false otherwise.
  * @returns {function} - The store enhancer (which adds event listeners to
  * dispatch actions on window resize).
  */
-export default (throttleTime = 100) =>
+export default (options) => {
```

Comment:
I know this is a really minor style change (sorry to be such a pain in the ass) but would you mind replacing this with argument deconstruction and default values? It really cleans up the logic, imo. What I had in mind was something like `export default ({throttleTime=100, calculateStateInitially = true})`.